### PR TITLE
Some README.md cleanups

### PR DESCRIPTION
**Pull request by miiichael**
Fix some typos and add a bunch of capitalisation, punctuation and newline tweaks. 😅

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -302,25 +298,25 @@ Since version 0.1.1 `launcherMeta` is now included, this can be used to get the
 
 ### /v2/versions/loader/:game_version/:loader_version/profile/json
 
-Returns the json file that should be used in the standard minecraft launcher
+Returns the JSON file that should be used in the standard minecraft launcher.
```
Comment (haykam821) [inline README.md:301]:
```suggestion
Returns the JSON file that should be used in the standard Minecraft launcher.
```

**Review by haykam821 — CHANGES_REQUESTED** [review]

**Comment by liach** [comment]
The sheer number of typos in this documentation is scary.

File: README.md [hunk 2]
```diff
@@ -302,25 +298,25 @@ Since version 0.1.1 `launcherMeta` is now included, this can be used to get the
...
 
-Returns the json file in format of the launcher json, but with the server's main class.
+Returns the JSON file in format of the launcher JSON, but with the server's main class.
 
 # V1
 
 ### /v1/versions
 
-full database, includes all the data, warning large json
+Full database, includes all the data. Warning, large JSON.
```
Comment (liach) [inline README.md:315]:
```suggestion
Full database, includes all the data. **Warning**: large JSON.
```

File: README.md [hunk 3]
```diff
@@ -1,26 +1,26 @@
...
+The versions are in order, the newest versions appear first.
 
 `game_version` and `loader_version` should be url encoded to allow for special characters. For example `1.14 Pre-Release 5` becomes `1.14%20Pre-Release%205`
 
 # V2
 
 ### /v2/versions
 
-full database, includes all the data, warning large json
+Full database, includes all the data. Warning, large JSON.
```
Comment (frqnny) [inline README.md:19]: **Warning** so its the same as liach’s thing.

**Review by frqnny — CHANGES_REQUESTED** [review]

**Comment by modmuss50** [comment]
I am literally blind to spelling mistakes, I try my best. And very much appreciate the help in fixing my mistakes 👍

**Comment by miiichael** [comment]
All good. I was originally intending to just correct a single spelling mistake. Nek minnit, all the fine tooth combs come out. 😅

**Review by modmuss50 — APPROVED** [review]

**Review by sfPlayer1 — APPROVED** [review]

**Review by frqnny — APPROVED** [review]

**Review by frqnny — APPROVED** [review]

**Review by liach — APPROVED** [review]

**Pull request merged**